src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.test.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import CheckboxGrid from './CheckboxGrid';
+import Checkbox from 'src/shared/checkbox/Checkbox';
+
+const checkboxOnChange = jest.fn();
+const gridData = {
+  default: {
+    symbol: {
+      id: 'symbol',
+      label: 'Symbol',
+      checkedStatus: false
```

Comment:
can we rename this field to `isChecked` instead of `checkedStatus`?

Reply:
That's done